scalableminds/webknossos
# backend refactoring: merge Project and ProjectSQL

this one was fairly easy since ProjectSQL and Project already had the same fields (no db lookup in jsonReads). Next up: Task

### Steps to test:
- create project
- create tasks using it, trace, finish some
- statistics should still be correct (open instance count)
- download it, should contain stuff
- view it as compoundproject, should contain stuff
- delete it


### Issues:
- contributes to  #2521

------
